pkg/ctl/delete/nodegroupd_test.go

```diff
@@ -0,0 +1,54 @@
+package delete
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
```

Comment:
Missing empty line between github imports and standard lib imports?

Reply:
oh yeah, you are right, my golangci-lint skipped test file :100: